cola/widgets/commitmsg.py

```diff
@@ -487,7 +487,7 @@ def commit(self):
         )
         self.context.runtask.start(
             task,
-            finish=lambda commit_task: self.commit_finished.emit(commit_task),
+            finish=self.commit_finished.emit,
```

Comment:
Thanks for these fixups! The `commit_finished` signal originally had a different signature but then it became the same and I missed the fact that I could drop the lambda.

Reply:
pylint did great work in this regard. I'm honestly astounded that it found this!